cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -201,15 +236,19 @@
       "items": {
         "properties": {
           "description": {
+            "description": "A brief description of the link",
             "type": "string"
           },
           "recid": {
+            "description": "If the link is to another Open Data record, the record id of that record",
```

Comment:
Can you update the case everywhere, e.g. `ID` instead of `id`, `URL` instead of `url`